intel-retail/retail-use-cases
# fix: resolve the yolov8n model downloading issue

fix: resolve the quantization issue of the yolov8n model downloading and uses openvino 2024.4 version compatible to run

## PR Checklist



## What are you changing?


## Issue this PR will close

close: **#45**

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable

- git clone this PR
- if there already existing yolov8 models under ./models/ directory, please clean it first: `rm -rf ./models/yolov8/`
- change directory to `models-downloader` and then run `./downloadOVMSModels.sh` and now you should see no quantization error any more.
- k8s helm capi-yolov8-ensemble also fixes towards to use this downloaded model
- 
## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/retail-use-cases )
